lib/ain-evm/src/evm.rs

```diff
@@ -250,16 +251,20 @@ impl EVMServices {
         }
 
         if ain_cpp_imports::past_changi_intermediate_height_4_height() {
+            let total_burnt_fees = U256::from(total_gas_used) * base_fee;
+            let total_priority_fees = total_gas_fees - total_burnt_fees;
             Ok((
                 *block.header.hash().as_fixed_bytes(),
                 failed_transactions,
-                total_gas_fees.try_into().unwrap(),
+                total_burnt_fees.try_into().unwrap(),
```

Comment:
Let's return these as Sats. Currently this looks to  be WEI.

Reply:
Thanks for this, I think the returned value is in GWEI. Will make the changes to return Sats, then we can nicely abstract gwei denomination to be only the EVM side of things.

Reply:
Resolved in ee66ff567